apps/vmq_metrics_plus/src/vmq_metrics_plus_sup.erl

```diff
@@ -0,0 +1,32 @@
+%%%-------------------------------------------------------------------
+%% @doc vmq_metric_plus top level supervisor.
+%% @end
+%%%-------------------------------------------------------------------
+
+-module(vmq_metrics_plus_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+
+-export([init/1]).
+
+-define(SERVER, ?MODULE).
+
+%% Helper macro for declaring children of supervisor
+-define(CHILD(I, Type), {I, {I, start_link, []}, permanent, 5000, Type, [I]}).
+
+start_link() ->
+    supervisor:start_link({local, ?SERVER}, ?MODULE, []).
+
+%% sup_flags() = #{strategy => strategy(),         % optional
```

Comment:
What is the use of these comments?

Reply:
removed [chore: remove unnecessary comments](https://github.com/gojek/vernemq/pull/53/commits/0bdbc9339d2fecb9926d8a968ca87b87658d8fca)